src/Tracing/TransactionContext.php

```diff
@@ -156,6 +158,21 @@ private static function parseTraceAndBaggage(string $sentryTrace, string $baggag
                 $context->parentSampled = $matches['sampled'] === '1';
                 $hasSentryTrace = true;
             }
+        } elseif (preg_match(self::W3C_TRACEPARENT_HEADER_REGEX, $sentryTrace, $matches)) {
+            if (!empty($matches['trace_id'])) {
+                $context->traceId = new TraceId($matches['trace_id']);
+                $hasSentryTrace = true;
+            }
+
+            if (!empty($matches['span_id'])) {
+                $context->parentSpanId = new SpanId($matches['span_id']);
+                $hasSentryTrace = true;
+            }
+
+            if (isset($matches['sampled'])) {
+                $context->parentSampled = $matches['sampled'] === '01';
```

Comment:
https://www.w3.org/TR/trace-context/#trace-flags

While this is a hex-encoded bit field, I think we can get away with just checking for `01`, as the other values are either `00` or omitting it altogether.